packages/server/src/unstable-core-do-not-import/parser.ts

```diff
@@ -110,5 +118,17 @@ export function getParseFn<TType>(procedureParser: Parser): ParseFn<TType> {
     };
   }
 
+  if ('~standard' in parser) {
+    // StandardSchemaEsque
+    return async (value) => {
+      const result = await parser['~validate']({ value });
+      if (result.issues) {
+        console.error('Schema issues:', result.issues);
+        throw new Error(result.issues[0].message);
```

Comment:
Should there not be a StandardError or something? Having a generic error here makes it impossible to do errorFormatter like this one? https://trpc.io/docs/server/error-formatting#adding-custom-formatting

Can we attach all the issues on Error.cause or something?

Reply:
I added our `SchemaError` utility class. Does this look okay to you? Or should we add `SchemaError` directly to the source code and not import it from `@standard-schema/utils`?

Reply:
Yea no deps and mports. Just a way to revive the full error with all the issues instead of just the first one